integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1392,6 +1393,18 @@ protected void initialize(Map<String, Object> inputDomainMap) throws Exception {
     TestUtils.exec(
         "cp -rf " + BaseTest.getProjectRoot() + "/kubernetes/samples " + BaseTest.getResultDir());
 
+    if (inputDomainMap.containsKey("customDomainTemplate")) {
```

Comment:
create a private method copyDomainTemplate() for this and call from callCreateDomainScript()

Reply:
done